### PR TITLE
Fix more multicpu stuff!

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

**Comment by mrtracy** [comment]
LGTM

File: storage/store.go [hunk 1]
```diff
@@ -568,7 +568,7 @@ func (s *Store) startGossip() {
 	ctx := s.Context(nil)
 	// Periodic updates run in a goroutine and signal a WaitGroup upon completion
 	// of their first iteration.
-	s.initComplete.Add(1)
+	s.initComplete.Add(2)
```
Comment (bdarnell) [inline storage/store.go]: I'm surprised this was a race. Don't both calls to `Add()` definitely happen before any call to `WaitForInit`?

**Comment by bdarnell** [comment]
LGTM

File: storage/store.go (hunk 1 above)
Comment (tamird) [inline storage/store.go]: I never observed this actually racing, so you're probably right. I noticed
it while chasing the feed bug and fixed it on instinct.

**Comment by tschottdorf** [comment]
LGTM

**Pull request merged**